src/components/attendee/InputSelect/InputSelect.module.scss

```diff
@@ -16,9 +20,9 @@
   padding: space.empty(0.33) 5em space.empty(0.33) space.empty(1);
   border-radius: space.empty(0.33) 0px 0px space.empty(0.33);
   box-sizing: border-box;
-  width: 80%;
+  width: 100%;
```

Comment:
[Q] Is there a need for `narrow` and `wide` variants (or differently named prop) of `InputSelect`?

Reply:
err, not now I think
so far it was alright using one variant with input's container determining the width